examples/ota-requestor-app/ota-requestor-common/ota-requestor-app.zap

```diff
@@ -1297,7 +1297,7 @@
               "mfgCode": null,
               "side": "server",
               "included": 1,
-              "storageOption": "External",
+              "storageOption": "NVM",
```

Comment:
Doing this means that the `OTARequestorStorage` will be bypassed

Reply:
This should not matter, because we should be forcing all list-typed attributes to external storage anyway, except someone broke it by doing a bad merge, apparently...  I'll get that fixed, and then this setting will go back to being ignored for this attribute.